src/app/app.component.html

```diff
@@ -23,5 +31,9 @@
 <app-math></app-math>
 <br>
 
+<app-sub-deals></app-sub-deals>
```

Comment:
Technically, should be at the end : - )

But I'll approve and merge so you can get the screenshot.